content/docs/command-reference/exp/run.md

```diff
@@ -24,9 +31,10 @@ Provides a way to execute and track <abbr>experiments</abbr> in your
 <abbr>project</abbr> without polluting it with unnecessary commits, branches,
 directories, etc.
```

Comment:
Wrt https://github.com/iterative/dvc.org/pull/4200#pullrequestreview-1227887123 @dberenbaum ,

> It explains some of the value.

Actually this is missing context because we don't mention anything else about Git in the refs so does this even make sense in `exp run` as-is now? Let me see if I can fix it (in both places)...

Reply:
Updated, PTAL.